tests/cs/cover_HassTurnOff.yaml

```diff
@@ -1,5 +1,52 @@
 language: cs
 tests:
+  - sentences:
+      - "zavři žaluzie ložnice"
```

Comment:
I hate here the name of fixture (which was of course not done by you), which makes here confusion between area command and entity name command. I will check on discord.

Reply:
Yes, it makes confusion however I have blind named just like that. I've prepared the PR because I was not able to close my blinds in the way I expected it should work.

Reply:
Not sure if it's right - now it requires to write sentences etc in the way that both (names and areas) are supported.